### PR TITLE
Enable translations for Adyen cronjob, useful for order confirmation …

### PR DESCRIPTION
**Pull request by lancergr**
…email sending for Magento 2.1

### CODE REVIEW

File: Model/Cron.php [hunk 1]
```diff
@@ -190,7 +198,8 @@ public function __construct(
         \Adyen\Payment\Model\Resource\Billing\Agreement\CollectionFactory $billingAgreementCollectionFactory,
         \Adyen\Payment\Model\Api\PaymentRequest $paymentRequest,
         \Adyen\Payment\Model\Order\PaymentFactory $adyenOrderPaymentFactory,
-        \Adyen\Payment\Model\Resource\Order\Payment\CollectionFactory $adyenOrderPaymentCollectionFactory
+        \Adyen\Payment\Model\Resource\Order\Payment\CollectionFactory $adyenOrderPaymentCollectionFactory,
+        ObjectManagerInterface $objectManager
```
Comment (rikterbeek) [inline Model/Cron.php]: better to use:
\Magento\Framework\App\AreaList $areaList

File: Model/Cron.php [hunk 2]
```diff
@@ -212,6 +222,12 @@ public function __construct(
      */
     public function processNotification()
     {
+        // needed for Magento < 2.2.0 https://github.com/magento/magento2/pull/8413
+        $areaList = $this->_objectManager->get(\Magento\Framework\App\AreaList::class);
```
Comment (rikterbeek) [inline Model/Cron.php]: then you can use:
$this->areaList->getArea(....)

Comment (rikterbeek) [inline Model/Cron.php]: we could add version check as well

**Comment by boldbart** [comment]
When will this be released? We're having issues with English order confirmations on Dutch store views.

**Pull request merged**